app/views/shared/export/_plan.erb

```diff
@@ -1,9 +1,30 @@
+<%
+  font_face = (@formatting[:font_face].present? ? "#{@formatting[:font_face]}" : 'Arial, Helvetica, Sans-Serif')
+  font_size = (@formatting[:font_size].present? ? "#{@formatting[:font_size]}" : '12')
+  margin_top = '5'
```

Comment:
if these values are meant to be defaults for exporting to pdf, I don't think here is the right place